cpp/docs_structure.bzl

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright (C) 2022 Vaticle
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+dir_mapping = {
+    "Transaction.adoc": "transaction",
+    "TransactionType.adoc": "transaction",
+    "QueryManager.adoc": "transaction",
+    "Explanation.adoc": "answer",
+    "Explainable.adoc": "answer",
+    "Explainables.adoc": "answer",
+    "ConceptMap.adoc": "answer",
+    "ConceptMapGroup.adoc": "answer",
+    "JSON.adoc": "answer",
+    "JSONType.adoc": "answer",
+    "ValueGroup.adoc": "answer",
+    "OwnerAttributePair.adoc": "answer",
+    "Future__RETURN__NATIVE_PROMISE__HELPER__.adoc": "common",
+    "Iterable__NATIVE_ITER__NATIVE_T__T__HELPER__.adoc" : "common",
+    "Iterator__NATIVE_ITER__NATIVE_T__T__HELPER__.adoc" : "common",
```

Comment:
Do these have to be so long? these will be rendered on the website how? Please check/render it to see what happens

Reply:
They will be long and ugly.
But in most places I've used a type alias, which should now be documented